### ISSUE TITLE
Multi-select wrap in tag bug

### ISSUE THREAD

**Issue by joshuaneedham** [sanusart/atom-wrap-in-tag#4]
I thank you for your contribution to Atom. When I tried using this package I ran into a bug that if it wasn't present would be a perfect solution for the elusive wrap multi-selection in tag for Atom.

```
US Toll Free:<br />
US Direct:<br />
<br />
Australia Direct:<br />
<br />
France Direct:<br />
 <br />
Germany Direct:<br />
<br />
India Toll Free:<br />
<br />
Netherlands Direct:<br />
<br />
Singapore Toll Free:<br />
<br />
United Kingdom Direct:<br />
<br />
Hong Kong Direct:<br />
```

I needed to select the text and wrap in a strong tag.

```
<strong>US Toll Free:</strong><br />
<strong>US Direct:</strong><br />
<br />
<strong>Australia Direct:</strong><br />
<br />
<strong>France Direct:</strong><br />
<br />
<strong>Germany Direct:</strong><br />
<br />
<strong>India Toll Free:</strong><br />
<br />
<strong>Netherlands Direct:</strong><br />
<br />
<strong>Singapore Toll Free:</strong><br />
<br />
<strong>United Kingdom Direct:</strong><br />
<br />
<strong>Hong Kong Direct:</strong><br />
```

However when I selected the text to be wrapped and added the tag to the multiple selections which worked by the way however it replaced the text as shown below.

```
<p>Hong Kong Direct:</p><br />
<p>Hong Kong Direct:</p><br />
<br />
<p>Hong Kong Direct:</p><br />
<br />
<p>Hong Kong Direct:</p><br />
<br />
<p>Hong Kong Direct:</p><br />
<br />
<p>Hong Kong Direct:</p><br />
<br />
<p>Hong Kong Direct:</p><br />
<br />
<p>Hong Kong Direct:</p><br />
<br />
<p>Hong Kong Direct:</p><br />
<br />
<strong>Hong Kong Direct:</strong><br />
```

It replaced all the items with the last text item and also only allowed the tag to be added to the last  item selected.

**Comment by sanusart** [comment]
@joshuaneedham Thanks for reporting, it is similar to #2 
The problem is that I did not took into account multiple selects, so the functionality flaw is in the core of this package, so stuff need to be rewritten. Hopefully, I'll get to rewrite it at some point in the near future. Sorry.

**Comment by sanusart** [comment]
Fixed by @FabioAntunes via 4c55dfb

**Comment by joshuaneedham** [comment]
Sweet! Thanks for fixing.
